src/app/Http/Requests/UpdateRefugeeRequest.php

```diff
@@ -26,18 +26,21 @@ public function authorize()
      */
     public function rules()
```

Comment:
I set the method to static and made the call

Reply:
Mmmh I prefer this way 

`$rules = (new StoreRefugeeRequest())->rules();`

(Because if you change to static, I would have to change some code elsewhere)

Reply:
Done ✅